source/configuration_manual/authentication/digest-md5.rst

```diff
@@ -0,0 +1,84 @@
+.. _authentication-digestmd5:
+
+==========
+Digest-MD5
+==========
+
+Digest-MD5 has two things that make it special and which can cause
+problems:
+
+-  Instead of using user@domain usernames, it supports **realms**.
+-  User name and realm are part of the MD5 hash that's used for
+   authentication.
+
+For these and other reasons `Digest-MD5 has been
+obsoleted <https://www.ietf.org/rfc/rfc6331.html>`_ by
+`SCRAM <https://www.ietf.org/rfc/rfc5802.html>`_.
+
+Realms
+^^^^^^
+
+Dovecot v1.0 has problems handling user@domain style usernames with
```

Comment:
This is some old information.. Should somehow rewrite this. Nowadays:
 * If username has no `@domain` and DIGEST-MD5 specifies realm, the username is converted into username@realm as the visible username.
 * Otherwise, the username is kept as it is. It's possible to use both user@domain style username and also use a realm.

Reply:
I rewrote the paragraph together with the "auth_default_realm" issue below to the best of my abilities, let me know if this is now incorrect.